packages/dts-generator/lib/phases/dts-code-gen.js

```diff
@@ -183,6 +183,9 @@ function genInterface(ast) {
   text += APPEND_ITEMS(ast.methods, genInstanceMethod);
 
   text += "}";
+  if (ast.namespace) {
```

Comment:
maybe add a comment that explains this added const export, e.g:
- with an example snippet
  ```typescript
        interface AbsoluteCSSSize {}
        export const AbsoluteCSSSize: AbsoluteCSSSize;
  ```
- explaining the why we do it

Because this is not trivial...